graph/telemetry/istio/appender/appender.go

```diff
@@ -114,9 +115,14 @@ const (
 	workloadListKey      = "workloadList"      // namespace vendor info
 )
 
-type serviceEntryHost struct {
+type serviceEntry struct {
 	location string
-	host     string
+	name     string // serviceEntry name
+}
+
+type serviceEntryHost struct {
+	host string
+	serviceEntry
```

Comment:
What does this declaration mean?
A variable named serviceEntry of type serviceEntry?

Reply:
Golang allows you to just provide a type name in a struct declaration.  The field name just becomes the same as the type name.  It is basically the same as saying:

serviceEntry serviceEntry

Reply:
Thanks!